node-graphql-react/packages/app/src/providers/ThemeProvider/ThemeProvider.js

```diff
@@ -0,0 +1,48 @@
+import {
+  createMuiTheme,
+  ThemeProvider as MuiThemeProvider,
+} from '@material-ui/core/styles';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import pink from '@material-ui/core/colors/pink';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { StylesProvider } from '@material-ui/styles';
+
+function ThemeProvider(props) {
+  const { children } = props;
+
+  const theme = React.useMemo(() => {
+    const nextTheme = createMuiTheme({
+      palette: {
+        background: {
+          default: '#F4F4F4',
```

Comment:
I think for this project it would make sense to either only use material ui colors (like with `pink`) or only hex colors. Maybe just hex to lower the barrier to entry (don't need to be as familiar with material specifically) unless that's what we're looking for.

Reply:
makes sense. Will update